serverless/jobs/how-to/reference-secret-in-job.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: How to reference secrets in Serverless Jobs
+  description: Steps to reference secrets from Secret Manager in your Serverless Jobs.
+content:
+  h1:  How to reference secrets in Serverless Jobs
+  paragraph: Steps to reference secrets from Secret Manager in your Serverless Jobs.
+tags: serverless jobs secrets secret-manager reference environment variable
+dates:
+  validation: 2024-10-27
+  posted: 2024-10-27
+categories:
+  - serverless
+  - jobs
+---
+
+Serverless Jobs seamlessly integrates with [Secret Manager](/identity-and-access-management/secret-manager/), which allows you to store, manage, and access sensitive information, such as credentials, SSH keys, SSL/TLS certificates, or any key/value pairs you need to secure.
+
+You can reference any secret stored in Secret Manager in a job, without having to hardcode any sensible data.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Created a Serverless Job](/serverless/jobs/how-to/create-job-from-scaleway-registry/)
+- [Created a secret](/identity-and-access-management/secret-manager/how-to/create-secret/)
+
+## Add a secret to a job
```

Comment:
```suggestion
## Reference a secret to a job
```

Reply:
or "in a job", if the verb is being changed to Reference